frontend/src/static/js/components/webstatus-overview-page.ts

```diff
@@ -130,16 +130,46 @@ export class OverviewPage extends LitElement {
       .then(allFeatures => {
         // Use CELL_DEFS to define the columns and
         // get the current (active) columns.
+        const columns: string[] = [];
         const columnKeys = parseColumnsSpec(getColumnsSpec(this.location));
-        const columns = columnKeys.map(
-          columnKey => CELL_DEFS[columnKey].nameInDialog
-        );
+
+        const pushBrowserName = (name: string) => {
+          columns.push(name);
+          const pattern =
+            /^Browser Implementation in (.+?)(?: (Experimental))?$/;
+          const match = name.match(pattern);
+          if (match) {
+            const browser = match[1];
+            const channel = match[2] ? 'Experimental' : 'Stable';
+            const wptName = `${browser} WPT ${channel} Score`;
+            columns.push(wptName);
+          }
+        };
+
+        columnKeys.forEach(columnKey => {
+          const name = CELL_DEFS[columnKey].nameInDialog;
+          switch (columnKey) {
+            case ColumnKey.Name:
+              columns.push(name);
+              break;
+            case ColumnKey.BaselineStatus:
+              columns.push(name);
+              break;
+            case ColumnKey.StableChrome:
+            case ColumnKey.StableEdge:
+            case ColumnKey.StableFirefox:
+            case ColumnKey.StableSafari:
+              pushBrowserName(name);
+              break;
```

Comment:
This is missing the cases for Experimental Column Keys

Reply:
Yes, since there was no fake data for this, I left this part off.  But generalizing and adding now.